LearnosityDemo/Pages/ItemsAPIDemo.cshtml.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using LearnositySDK.Request;
+using LearnositySDK.Utils;
+// static LearnositySDK.Credentials;
```

Comment:
this would be good to fix

Reply:
Fixed: https://github.com/Learnosity/learnosity-sdk-asp.net/commit/b81074bd4af12796fe00b45f67c0700d1f686d5d